BUILDING.md

```diff
@@ -83,8 +83,10 @@ Prerequisites:
 
 On macOS, you will also need:
 * [Xcode](https://developer.apple.com/xcode/download/)
```

Comment:
Can we reword this a bit? At the moment it says "On macOS, you will also need -> Xcode -> You also need to install the `Xcode Command Line Tools` ...".

A full installation of Xcode is not required, so maybe this can be changed to 

On macOS, you will also need:

- The `Xcode Command Line Tools` ...